solr/core/src/java/org/apache/solr/handler/admin/ShowFileRequestHandler.java

```diff
@@ -195,54 +195,69 @@ private void showFromZooKeeper(
   }
 
   // Return the file indicated (or the directory listing) from the local file system.
-  private void showFromFileSystem(SolrQueryRequest req, SolrQueryResponse rsp) {
+  private void showFromFileSystem(SolrQueryRequest req, SolrQueryResponse rsp) throws IOException {
     Path admin = getAdminFileFromFileSystem(req, rsp, hiddenFiles);
 
     if (admin == null) { // exception already recorded
       return;
     }
-    // TODO SOLR-8282 move to PATH
-    File adminFile = admin.toFile();
+    Path adminFile = admin;
```

Comment:
maybe just call `admin` `adminFile` in the first place?   Now that you fixed the todo?